src/scales/scale.ts

```diff
@@ -122,19 +118,23 @@ module Plottable {
       }
     }
 
-    protected _getDomain() {
-      return this._d3Scale.domain();
+    protected _getDomain(): D[] {
```

Comment:
I don't think you need the return type in the function definition. Check the `plottable.d.ts` if it is autogenerated anyway